### ISSUE TITLE
UT_XUNIT_REPORTER format not conform https://gist.github.com/kuzuha/232902acab1344d6b578

### ISSUE THREAD

**Issue by diermen13** [utPLSQL/utPLSQL#634]
In the ut_xunit_reporter spec it is specified that the XML generated is in a format conforming with JUnit4.
Looks like that's not the case. 
E.g. the attributes in node testsuites: failure, error and skipped should be resp. failures, errors and disabled.

**Comment by lwasylow** [comment]
I think it was written mainly to integrate into jenkins, and name is a bit confusing. Are you using any specific CI server to publish results ?

**Comment by diermen13** [comment]
I'm using Jenkins.

**Comment by jgebal** [comment]
It works perfectly fine with Jenkins junit test results publishing

**Comment by diermen13** [comment]
Yep. I publish now with JUnit test result, but that's in fact not the issue...

**Comment by viktors1982** [comment]
The JUnit4 ir valid for xunit junit:
![image](https://user-images.githubusercontent.com/1754235/38556305-e074c09c-3cd1-11e8-918a-c969336d77d6.png)

But JUnit4 is not valid for 
![image](https://user-images.githubusercontent.com/1754235/38556368-1cd4c73a-3cd2-11e8-9f59-5da512483532.png)

And Junit4 not is UT_XUNIT_REPORTER

**Comment by diermen13** [comment]
> And Junit4 not is UT_XUNIT_REPORTER

Exactly. 
But this is what's specified in the specs:
/**
   * The XUnit reporter.
   * Provides outcomes in a format conforming with JUnit4 as defined in:
   *  https://gist.github.com/kuzuha/232902acab1344d6b578
   */

**Comment by lwasylow** [comment]
Will look into getting that in line with junit4 either to update or if breaking into new one.

**Comment by jgebal** [comment]
Can you point to JUnit 4 standard reference?

**Comment by diermen13** [comment]
Its a pretty simple adjustment.
get_common_item_attributes, replace skipped with disabled and add an 's' to error and failure
finally skip the call get_common_item_attributes in print_test_elements

**Comment by jgebal** [comment]
Looks like you're right @jgarec. We didn't spot this issue before as the format worked fine with some of Jenkins test results plugins.

We should definitely get this fixed.

Did you check that the format is consumed without issues after making the adjustments/fixes you sugest or is it just a guess based on the difference between defined format and outputs?

**Comment by jgarec** [comment]
See [Junit xsd used by jenkinsci/xunit-plugin](https://github.com/jenkinsci/xunit-plugin/tree/master/src/main/resources/org/jenkinsci/plugins/xunit/types/model/xsd)
